manifests/service.pp

```diff
@@ -42,6 +42,7 @@
       },
       enable   => $teleport::service_enable,
       provider => $init_style,
+      subscribe => [Exec['extract-teleport-current'], File["${teleport::bin_dir}/tctl"], File["${teleport::bin_dir}/teleport"], File["${teleport::bin_dir}/tsh"]],
```

Comment:
Do we need to subscribe to tctl or tsh? Those aren't used by this service, right?

Reply:
Tests leftovers, I'll remove them 👍